services/mailer.js

```diff
@@ -0,0 +1,43 @@
+const nodemailer = require('nodemailer');
+const sgTransport = require('nodemailer-sendgrid-transport');
+const debug = require('debug')('talk:mail');
+const options = {
+  auth: {
+    api_key: process.env.TALK_SENDGRID_APIKEY
```

Comment:
Please use an SMTP_CONNECTION_STRING var instead of custom sendgrid code. This will allow us to connect to just about any email service.